packages/mdc-drawer/README.md

```diff
@@ -68,6 +68,8 @@ However, you can also use SVG, [Font Awesome](https://fontawesome.com/), or any
 ```scss
 @import "@material/drawer/mdc-drawer";
 @import "@material/list/mdc-list";
+// For dismissible drawer, include mdc-side-sheet for the shared mdc-side-sheet-app-content class
```

Comment:
Hm. Actually we could just genericize that heading to just "Icons" since the drawer items each have an icon, too, and the note is equally applicable to that.

Reply:
Sent #3990 to fix this in master.